plugins/t/check_curl.t

```diff
@@ -46,7 +46,7 @@ $res = NPTest->testCmd(
 	);
 cmp_ok( $res->return_code, '==', 2, "Webserver $host_nonresponsive not responding" );
 # was CRITICAL only, but both check_curl and check_http print HTTP CRITICAL (puzzle?!)
-cmp_ok( $res->output, 'eq', "HTTP CRITICAL - Invalid HTTP response received from host on port 80: cURL returned 28 - Timeout was reached", "Output OK");
+cmp_ok( $res->output, 'eq', "HTTP CRITICAL - Invalid HTTP response received from host on port 80: cURL returned 28 - Connection timed out after 3001 milliseconds", "Output OK");
```

Comment:
I guess it would be better to do a `like()` here as well, testing for exactly 3001 milliseconds is probably a bad idea.

Reply:
@sni Yep agreed, fixed that now & tests are passing locally.